tests/config/functional/setup_cfg/issue_4272/option_in_wrong_section.cfg

```diff
@@ -0,0 +1,6 @@
+# Test for the behavior of https://github.com/PyCQA/pylint/issues/4272
+[pylint.MASTER]
+load_plugins=pylint.extensions.bad_builtin
+persistent=no
+[pylint.DEPRECATED_BUILTINS]
+bad-functions=print,map
```

Comment:
Why is bad-functions unrecognised? It's defined in `deprecated_builtins`?

Reply:
Hmm, I think you detected a problem in our own pylintrc, because it's in this section for our pylintrc but it's an option of the ``BadBuiltinChecker``. I wonder why it's not raising an error then.

Reply:
Shouldn't it be in the `.json` output?

Reply:
Yes, but the value for ``bad-functions`` is not loaded and the test fails. I'm investigating right now.

Reply:
The baseline for the default configuration without change do not load any plugins, so no extensions and no extensions' options. There's an easy fix (loading all plugin when creating the default conf) but it prevents testing the load-plugin option. I think I'm going to permit to define a baseline configuration alongside existing files.

Reply:
Hmm, what if we iterate over the updated config keys instead of over the default config? Wouldn't we see any new keys introduced?

Reply:
We create the expected values from the default config we create in the tests modified with result.json. The issue we have is when creating the expected result for configuration where some non-default plugin should be loaded. If we iterate over the updated config how would we know what is the expected value ?

Reply:
Currently we do `default config - final config == expected json` right? Can't we do `final config - default config == expected json`? If there is a key in `final` that is not in `default` it should be completely equal to the value in the `json`. That way we can handle unexpected keys right?

Reply:
Right, I think it could work, let me try.